runtime/runtime_test.go

```diff
@@ -118,3 +120,18 @@ func setStorageAccountInfo(t *testing.T, storage *runtime.Storage, account []byt
 
 	return keyStorageAccount, accountInfo
 }
+
+func getQueryInfo(t *testing.T, runtime *wasmer.Instance, extrinsic []byte) primitives.RuntimeDispatchInfo {
+	buffer := &bytes.Buffer{}
+
+	buffer.Write(extrinsic)
+	sc.U32(buffer.Len()).Encode(buffer)
```

Comment:
Doesn't the extrinsic come with an already encoded length?

Reply:
The API has a separate argument, specific for the length